### DISCUSSION TITLE
[1888160] ARMORED CORE™ VI FIRES OF RUBICON™  [仅本体]

### DISCUSSION THREAD

**Discussion by RieKKA** [pjy612/SteamManifestCache#505] [Share]
`appid:1888160`
keys:
```
1888161;2cbbae38e8fbb1cffbba8ca059d96e8b2ee29f77fdce9a9f47e666d333366341
1888164;347a0ccf0383122fcfc3fc32a9e06d397e3b39da49a87e3404dae449bac0b8bf
```
内测码:5513A2912F545035
邮箱:[x2804084975@hotmail.com]
[1888161_704565332477559849.zip](https://github.com/pjy612/SteamManifestCache/files/14356065/1888161_704565332477559849.zip)

**Comment by CSSZYF** [comment]
#26 缺少清单文件

**Reply by RieKKA** [reply]
已上传

**Comment by pjy612** [comment]
* 密钥格式不对 #26
 * 标题id错误

**Reply by RieKKA** [reply]
已修改

**Reply by RieKKA** [reply]
上面那个邮箱好像接收不到邮件了 新邮箱地址 3416837474@qq.com

**Reply by pjy612** [reply]
最好是你github绑定的邮箱

**Reply by RieKKA** [reply]
后面发的qq邮箱就是

**Comment by barryhamsy** [comment]
too bad this is missing 1888163 for the armoredcore6.exe files.

**Reply by barryhamsy** [reply]
If anyone need the game in full, download this and extract, drag and drop to steamtools, enjoy!
[Armored Core 6.zip](https://github.com/user-attachments/files/16446175/Armored.Core.6.zip)
